app/human-detector-app/config/ServerConfig.tsx

```diff
@@ -6,3 +6,4 @@ export const getSendNotifKeyUrlExtension = (userId: string): string =>
 export const getGroupsListUrlExtension = (userId: string): string => `/users/${userId}/groups`;
 export const getNotificationHistoryUrlExtension = (userId: string): string =>
   `/users/${userId}/notifications`;
+export const exchangeCodeForTokenUrlExtension = '/auth/userAccessToken';
```

Comment:
We scrapped this endpoint, so this can be removed

Reply:
scrapped endpoint changes done 2b640adf45439034c1d77965228d7fc0e1972115